apps/web/contexts/common-contexts.ts

```diff
@@ -4,6 +4,11 @@ import {createClient} from '@supabase/supabase-js'
 export const supabase = createClient(
   'https://hhnxsazpojeczkeeifli.supabase.co',
```

Comment:
Thought you added an env var for supabase url?

Reply:
In another branch, but it was broken on every env except prod so I didn't roll it in for now. Feel free to tweak or I will soon